Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/EcLiteBat0Virt.asl

```diff
@@ -0,0 +1,159 @@
+/** @file
+  ACPI DSDT table
+
+  Copyright (c) 2016 - 2021, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+Scope (\_SB)
+{
+  Name(B0SC, 90) // Battery 0 Stored Capacity
+  Name(ECPR, 0) // Eclite Present or not
+
+  // Battery Test Method:
+  //    This method updates the Virtual Battery percentages and
+  //    notifies the OS of these changes.
+  //
+  //  Arguments:
+  //    Arg0: The desired Virtual Battery Percentage.
+  //
+  //  Return Value:
+  //    None
+  Method(BTTM,1,Serialized)
+  {
+    If(PWRS)
+    {
+      If(LGreaterEqual(Arg0,B0SC))
+      {
+        Store(Arg0,B0SC)
+        Notify(BAT0,0x80)
+      }
+    }
+    Else
+    {
+      If(LLessEqual(Arg0,B0SC))
+      {
+        Store(Arg0,B0SC)
+        Notify(BAT0,0x80)
+      }
+    }
+  }
+
+  //
+  // Method to Handle AC/DC Virtual Switch
+  //
+  Method (ADCS, 1, Serialized) {
+    Store (0xF, Local0)
+    //
+    // Read AC/DC Switch GPIO RX State
+    // Expecting either value of 0 or 1.
+    //
+    If (CondRefOf (\_SB.GGIV)) {
+      Store (\_SB.GGIV(\VBSG), Local0)
+    }
+
+    If (LEqual (Local0, 0xF)) {
+      Return ()
+    }
+
+    If (LAnd(ECPR, LEqual(\_SB.ECLT.BNUM, 0))) {
+      If (LNot (Local0)) {
+        Store(ECLITE_DC_PRESENT,PWRS)
+      } Else {
+        Store(ECLITE_AC_PRESENT,PWRS)
+      }
+
+      If(Arg0) {
+        // Perform needed ACPI Notifications.
+        Notify(BAT0,0x80)
+      }
+    }
+    Return ()
+  }
+
+  // Define the Virtual Battery 0 Control Method.
+  Device(BAT0)
+  {
+    Name(_HID,EISAID("PNP0C0A"))
+    Name(_UID,0)
+
+    Method(_STA,0)
+    {
+#ifdef ECLITE_PSE_SUPPORT

```

Comment:
Is this defined or not - ECLITE_PSE_SUPPORT?

Reply:
This is the mistake during porting. Will remove the ifdef condition.